server/src/app.js

```diff
@@ -1,8 +1,15 @@
+import 'dotenv/config';
 import express from 'express';
+import mongoose from 'mongoose';
 import apiRouter from './routes';
 
 const app = express();
 
+mongoose.connect(process.env.DB_CONNECT, (err) => {
```

Comment:
I would put it in the async function (could be stored in configuration) and call it here await in result (either using top level await or wrapping all app.js code with async function).

Inside the function in case connection cannot be established (in the catch block) I think we can even exit proccess as there is no reason to start the whole app without working database connection